pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
ah, then use the existing VolumeGidAnnotationKey const... I thought this was net new